erpnext/support/doctype/issue/issue.py

```diff
@@ -212,7 +212,143 @@ def make_issue_from_communication(communication, ignore_communication_links=Fals
 
 	return issue.name
 
+def get_time_in_timedelta(time):
+	"""
+		Converts datetime.time(10, 36, 55, 961454) to datetime.timedelta(seconds=38215)
+	"""
+	return timedelta(hours=time.hour, minutes=time.minute, seconds=time.second)
+
+def set_first_response_time(communication, method):
+	parent = get_parent_doc(communication)
+	if parent.doctype == "Issue":
```

Comment:
```suggestion
	if communicaton.get('reference_doctype') == "Issue":
		parent = get_parent_doc(communication)
```

Reply:
On every communication, we can prevent a `get_doc` to be executed

Reply:
That sounds much better, I'll do that.